### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 0701pawan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ Core ladder/project functionality
 Web API endpoints and functionality
 
 ### aiarena/frontend
-Django template website frontend
+Django template is used for website frontend
```
Comment (lladdy) [inline README.md:16]:
```suggestion
Django templates are used for the website frontend
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # aiarena-web
-A website for running the aiarena.net ladder.
+This is an amazing website which gives you the sense of thrill unables you to compete with other bots across the world
```
Comment (lladdy) [inline README.md:2]:
```suggestion
This is an amazing website which gives you a sense of thrill and enables you to compete with other bots across the world 
```

**Review by lladdy — CHANGES_REQUESTED** [review]
Hi @0701pawan,
Thanks for your contribution! :)
I have submitted some change requests for you to review.
Let me know if you have any thoughts.

**Comment by 0701pawan** [comment]
Thank you 
I think adding pictures to the readme would make it more beautiful.

**Comment by lladdy** [comment]
> 
> 
> Thank you
> I think adding pictures to the readme would make it more beautiful.

@0701pawan 
I very much agree. Do you have any suggestions for pictures we might add?

**Comment by 0701pawan** [comment]
I think screenshots from the website would work

**Comment by 0701pawan** [comment]
I have added some images  please have a look

**Comment by 0701pawan** [comment]
Just a gentle reminder I  added some images in the readme have a look

**Comment by eladyaniv01** [comment]
> Just a gentle reminder I added some images in the readme have a look

Thank you so much for your work @0701pawan,
We are aware of it and appreciate it dearly,  please be patient while we find the time to review it properly.

**Comment by 0701pawan** [comment]
thankyou

**Comment by lladdy** [comment]
Hi @0701pawan,
You'll need to review what the changes look like.
There are two images separated by a white line and they have watermarks from an un-activated windows on them.

**Comment by danielvschoor** [comment]
Closing due to no activity in more than 6 months

**Pull request closed without merging**